src/components/NewCommentForm.tsx

```diff
@@ -1,12 +1,14 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-import { CommentData } from '../types/Comment';
-
+import { useAppDispatch } from '../app/hooks';
+import { addComment } from '../features/comments/сommentsSlice';
```

Comment:
There is a typo in the import path for `addComment`. The directory name `сommentsSlice` seems to be misspelled. Ensure that the path is correct to avoid import errors.